apps/graph/list/list_controller.h

```diff
@@ -21,6 +21,8 @@ class ListController : public Shared::FunctionListController, public Shared::Tex
   bool textFieldDidAbortEditing(TextField * textField) override;
   bool textFieldShouldFinishEditing(TextField * textField, Ion::Events::Event event) override;
   bool textFieldDidReceiveEvent(TextField * textField, Ion::Events::Event event) override;
+protected:
+  virtual const char * recordExtension() const override { return Ion::Storage::funcExtension; }
```

Comment:
The build bug is from the override keyword here :
```suggestion
  virtual const char * recordExtension() const { return Ion::Storage::funcExtension; }
```